map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -74,24 +76,29 @@
   @FXML
   private void initialize() {
 
-    // set basemap options
-    basemapList.getItems().addAll(Basemap.createStreets(), Basemap.createImagery(), Basemap
-            .createTopographic(), Basemap.createOceans());
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-    // update basemap when selection changes
-    basemapList.getSelectionModel().select(0);
-    basemapList.getSelectionModel().selectedItemProperty()
-            .addListener(o -> map.setBasemap(basemapList.getSelectionModel().getSelectedItem()));
+    // set the basemap style options
+    basemapStyleList.getItems().addAll(BasemapStyle.ARCGIS_STREETS, BasemapStyle.ARCGIS_IMAGERY_STANDARD,
+      BasemapStyle.ARCGIS_TOPOGRAPHIC, BasemapStyle.ARCGIS_OCEANS);
```

Comment:
Small formatting suggestion, but up to you. I think these things look near when each entry is on it's own line, as I find it a lot easier to read:
```suggestion
    basemapStyleList.getItems().addAll(
	    BasemapStyle.ARCGIS_STREETS, 
	    BasemapStyle.ARCGIS_IMAGERY_STANDARD,
	    BasemapStyle.ARCGIS_TOPOGRAPHIC, 
	    BasemapStyle.ARCGIS_OCEANS);
```